doc/source/nexus.rst

```diff
@@ -0,0 +1,13 @@
+Nexus integration
```

Comment:
I think we should write up some docs on how this works, the architecture, how to register tools, how to make news ones, where to put them, etc

Reply:
I added a new page in docs for Nexus integration where I clarified this.